Ladybird/Qt/ladybird.qrc

```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE RCC><RCC version="1.0">
     <qresource>
-        <file>../Icons/ladybird.png</file>
```

Comment:
This feels pretty weird. Would it be better to copy the icon over the old name instead?

Reply:
I could symlink the two perhaps

Reply:
Is this OK?

Reply:
Hmmm I suppose. But in theory, we should provide more than one icon size. 

We're not quite using QIcon directly, I think, by providing it only one QPixmap.
<https://doc.qt.io/qt-6/qicon.html#high-dpi-icons>

Ideally we'd follow the FreeDesktop Icon Theme spec and provide an icon theme for our app that checks all the boxes. <https://specifications.freedesktop.org/icon-theme-spec/icon-theme-spec-latest.html>

Reply:
Made an issue for this: #643